qgreenland/config/helpers/steps/compress_and_add_overviews.py

```diff
@@ -15,14 +15,21 @@
     'average_magphase',
     'mode',
 ]
+CompressionType = [
+    'DEFLATE',
+    'JPEG',
+]
 
 
 def compress_and_add_overviews(
     *,
     input_file: str,
     output_file: str,
+    # TODO: This parameter is ignored if the compression type isn't deflate.
+    # Refactor args?
```

Comment:
We should make `dtype_is_float` `Optional`. Raise an error if it's given with a `compression_type` other than `DEFLATE`. Raise an error if it's not given with a `compression_type` of `DEFLATE`.

Reply:
032c021